packages/opentelemetry-shim-opentracing/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@opentelemetry/shim-opentracing",
+  "version": "0.0.1",
+  "description": "OpenTracing to OpenTelemetry shim",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.ts",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "clean": "rimraf build/*",
+    "check": "gts check",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "posttest": "yarn run check"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "tracing",
+    "profiling"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.7",
```

Comment:
Could you add a `tslint.json` file that matches https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-core/tslint.json ? And then also add the `tslint-microsoft-contrib` package? That will enable this package to pick up new TSLint goodies such as the license header checks, and anything else we add in later

Reply:
also please add "tslint-consistent-codestyle":"^1.15.1",